pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -46,6 +49,19 @@ func minioEnvironmentVars(t *miniov1.Tenant) []corev1.EnvVar {
 	}, corev1.EnvVar{
 		Name:  "MINIO_UPDATE_MINISIGN_PUBKEY",
 		Value: "RWTx5Zr1tiHQLwG9keckT0c45M3AGeHD6IvimQHpyRywVWGbP1aVSGav",
+	}, corev1.EnvVar{
+		Name: "MINIO_ARGS",
+		Value: fmt.Sprintf("%s://%s:%s@%s:%s%s/%s/%s",
```

Comment:
This entire string can be stored in a secret and read in as an environment variable.

Reply:
Yes definitely, I didn't add more code - simply wanted to make it available for testing.  Please test  this first we can make it more stricter soon..

Reply:
Not sure what we achieve by creating a secret.

Reply:
I thought the intention is just to avoid it leaking upon kubectl describe pod

Reply:
In a deployment where namespace is a proxy for tenancy, any user with read permissions on Pods can know the API endpoint for operator and credentials to use. There is no good reason to not secure it.

Reply:
> In a deployment where namespace is a proxy for tenancy, any user with read permissions on Pods can know the API endpoint for operator and credentials to use. There is no good reason to not secure it.

:+1:

Reply:
We can do this later, going to resolve this right now.

Reply:
I have the commit ready, can push it after testing...